From kay.sievers@vrfy.org  Thu Sep 14 02:27:10 2006
Subject: CONFIG_SYSFS_DEPRECATED_FUTURE
From: Kay Sievers <kay.sievers@vrfy.org>
To: greg@kroah.com
Date: Thu, 14 Sep 2006 11:23:28 +0200
Message-Id: <1158225809.3167.8.camel@min.off.vrfy.org>

Turn off sysfs features that will be removed in the future

Signed-off-by: Kay Sievers <kay.sievers@vrfy.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 init/Kconfig |    9 +++++++++
 1 file changed, 9 insertions(+)

--- a/init/Kconfig
+++ b/init/Kconfig
@@ -302,6 +302,15 @@ config SYSFS_DEPRECATED
 
 	  If you are unsure about this at all, say Y.
 
+config SYSFS_DEPRECATED_FUTURE
+	bool "Create sysfs files which will be deprecated in the future"
+	default y
+	help
+	  This option creates the compatibility /sys/class, /sys/block
+	  and /sys/bus directories.
+	  Removing these directories is useful for testing software,
+	  which should only depend on the unified layout at /sys/subsystem.
+
 config RELAY
 	bool "Kernel->user space relay support (formerly relayfs)"
 	help
